templates/inspec/examples/google_spanner_instance/google_spanner_instance.erb

```diff
@@ -0,0 +1,11 @@
+<% gcp_project_id = "#{external_attribute('gcp_project_id', doc_generation)}" -%>
+<% spannerinstance = grab_attributes['spannerinstance'] -%>
+
+describe google_spanner_instance(project: <%= gcp_project_id -%>, name: <%= doc_generation ? "'#{spannerinstance['name']}'" : "spannerinstance['name']" -%>, config: <%= doc_generation ? "'#{spannerinstance['config']}'" : "spannerinstance['config']" -%>) do
+  it { should exist }
+  its('config') { should eq <%= doc_generation ? "'#{spannerinstance['config']}'" : "spannerinstance['config']" -%> }
+  its('name') { should eq <%= doc_generation ? "'#{spannerinstance['name']}'" : "spannerinstance['name']" -%> }
+  its('display_name') { should eq <%= doc_generation ? "'#{spannerinstance['display_name']}'" : "spannerinstance['display_name']" -%> }
+  its('node_count') { should eq <%= doc_generation ? "'#{spannerinstance['node_count']}'" : "spannerinstance['node_count']" -%> }
+  its('labels') { should eq <%= doc_generation ? "'#{spannerinstance['labels']}'" : "spannerinstance['labels']" -%> }
```

Comment:
This should use the InSpec map include form, similar to https://github.com/GoogleCloudPlatform/magic-modules/blob/c1cebaf1374b73ad7ee5cf660ec29fa2cfe5d6d8/templates/inspec/examples/google_dataproc_cluster/google_dataproc_cluster.erb#L6

Essentially you can describe a key, value pair as with `include(key, value)` and it will ensure that pair is in the map

Reply:
ya. it make sense. Let me add.